unittests/Dialect/HW/GraphFixture.h

```diff
@@ -0,0 +1,21 @@
+//===- GraphFixutre.h - A fixture for instance graph unit tests -----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// NOLINTNEXTLINE(llvm-header-guard)
+#ifndef UNITTED_DIALECT_FIRRTL_HW_GRAPHFIXTURE_H
+#define UNITTED_DIALECT_FIRRTL_HW_GRAPHFIXTURE_H
+
+#include "circt/Dialect/HW/HWOps.h"
+
+namespace fixtures {
+
+mlir::ModuleOp createModule(mlir::MLIRContext *context);
+
+} // end namespace fixtures
+
+#endif // UNITTED_DIALECT_FIRRTL_HW_GRAPHFIXTURE_H
```

Comment:
Macro here as well, also EOL.  Consider configuring editor to do this for you.

Reply:
clang-tidy should do it, but I think there is something borked with the rule with the versions I'm using :(. Thanks for spotting this!